projects/transformers/run_utils.py

```diff
@@ -921,7 +1271,7 @@ def check_best_metric(training_args, task_name, metric=None):
 
     print("metric configuration after checks: "
           f"{metric}, {greater_is_better}")
-    training_args.greater_is_beter = greater_is_better
+    training_args.greater_is_better = greater_is_better
```

Comment:
That is a weird argument - shouldn't this just be a function of the type of metric used?

Reply:
Pretty much, yeah. In HF it is an argument. In our code, it's an argument and these checks are supposed to ensure the correct configuration. Ideally I agree it isn't an argument at all but simply something determined on the fly based on the metric.